apps/hosts/utils.py

```diff
@@ -1,6 +1,13 @@
 from .models import ChallengeHost
+from challenges.models import Challenge
 
 
 def get_challenge_host_teams_for_user(user):
     """Returns challenge host team ids for a particular user"""
     return ChallengeHost.objects.filter(user=user).values_list('team_name', flat=True)
+
+
+def is_user_a_host_of_challenge(user, challenge_pk):
```

Comment:
The function and doc string does not match. Can you please check and update them as per the function defintion?

As per my understanding, function name is correct.